src/libraries/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -337,9 +338,9 @@ public sealed partial class ImmutableDictionary<TKey, TValue> : System.Collectio
         System.Collections.Immutable.IImmutableDictionary<TKey, TValue> System.Collections.Immutable.IImmutableDictionary<TKey,TValue>.SetItems(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<TKey, TValue>> items) { throw null; }
         public System.Collections.Immutable.ImmutableDictionary<TKey, TValue>.Builder ToBuilder() { throw null; }
         public bool TryGetKey(TKey equalKey, out TKey actualKey) { throw null; }
```

Comment:
[MaybeNullWhen(false)] on the out?

Reply:
When actualKey is not found we not setting null or default, setting equalKey to it, the method implementation:
```csharp
private static bool TryGetKey(TKey equalKey, MutationInput origin, out TKey actualKey)
{
    int hashCode = origin.KeyComparer.GetHashCode(equalKey);
    HashBucket bucket;
    if (origin.Root.TryGetValue(hashCode, out bucket))
    {
        return bucket.TryGetKey(equalKey, origin.Comparers, out actualKey);
    }

    actualKey = equalKey;
    return false;
}
```

Reply:
I see. Ok.